### DISCUSSION TITLE
Compatible Xcode Versions

### DISCUSSION THREAD

**Discussion by xkittyqueenx** [cryptoAlgorithm/Swiftcord#2] [General]
With what Xcode version is this project compatible?

**Comment by cryptoAlgorithm** [comment]
I've only tested it on >13.1, I cannot verify if it works on previous versions. One file wasn't added to git previously, I just made a commit that should fix that.

**Reply by cryptoAlgorithm** [reply]
I've tested it from 13.1-13.2.1 on the arm64 arch, builds fine on all versions in that range
